### DISCUSSION TITLE
Z2MQTT does not accept IP change of coordinator

### DISCUSSION THREAD

**Discussion by canoben** [Koenkk/zigbee2mqtt#23677] [Devices]
I'm using a SLZB-06M coordinator over Ethernet with Home Assistant. Recently I changed the IP address used by the device. But whatever I try (restarting the integration, uninstalling and reinstalling; tried several times) Z2MQTT continues to use the **old** IP address and errors out.

In the configuration I've put the new address (x.x.x.141):
<img width="320" alt="Capture d’écran 2024-08-20 à 19 49 59" src="https://github.com/user-attachments/assets/750f22f7-dd3c-4a71-8051-1504b732def9">

But in the log the old address still appears every time (x.x.x.218)
<img width="516" alt="Capture d’écran 2024-08-20 à 19 50 17" src="https://github.com/user-attachments/assets/4fd5c692-de0b-4c58-b04e-0f8cacb75021">

What am I missing here?

**Comment by EraseSURE** [comment]
Hmm - did you delete the old lease? Are you sure the SLZB is using the new 141 address?

Try stopping the integration - editing the config with the new address again - save - then restart

**Reply by EvertDekker** [reply]
Isn't it that this setup uses **ser2net** and maybe this services need to be restarted or the ser2net.conf need to be changed.

**Comment by Smith166** [comment]
I have he same, I tried everything. (actually not everything I know... I didn't try to solution yet... :( )

**Reply by chris-1243** [reply]
- What kind of installation do you use ?
- Where did you edit the change ?

And please share some log info as well (debug)
